Test-NetStack.psm1

```diff
@@ -156,7 +160,17 @@ Function Test-NetStack {
         [Parameter(Mandatory = $false, ParameterSetName = 'OnlyPrereqIPTarget'   , position = 4)]
         [Parameter(Mandatory = $false, ParameterSetName = 'RevokeFWRulesNodes'   , position = 4)]
         [Parameter(Mandatory = $false, ParameterSetName = 'RevokeFWRulesIPTarget', position = 4)]
+	[Parameter(Mandatory = $false, ParameterSetName = 'Experimental'         , position = 4)]
         [Switch] $Experimental = $false,
+        
+	[Parameter(Mandatory = $false, ParameterSetName = 'Experimental'      , position = 5)]
+        [String] $DpdkUser='',
```

Comment:
I thought we were doing away with the user here since it required a specific setup beforehand anyway?

Reply:
yes you're right, forgot to remove that.